### DISCUSSION TITLE
Add support for custom separators in CSV attachments (semicolon)

### DISCUSSION THREAD

**Discussion by lubomsky** [allure-framework/allure2#2733] [Ideas & Feedback]
Hello, any updates/info about rendering csv preview in allure report when `;` is used as a separator?
https://github.com/allure-framework/allure1/pull/526#issuecomment-87491863

Is there any solution to see it properly like here:
![csv table](https://cloud.githubusercontent.com/assets/9350353/6635979/423c6ee4-c974-11e4-905a-1e09a367d129.png)

Regards

**Comment by baev** [comment]
Allure 1 reached EOL a long time ago. Please update to the latest version of Allure 2 (2.30.0 as of today).

**Reply by lubomsky** [reply]
Sorry for not being precise.
The linked topic is about allure1, but the question is still active regarding allure2 which I am using right now (java version).

**Reply by baev** [reply]
Ah, I see. I checked the source code, and it seems like Allure does not support semicolon-separated values. I'll reopen the discussion and transfer it into a feature-request category.
